.github/workflows/nightly.yml

```
@@ -16,23 +16,20 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - distro: foxy
-            branch: foxy
-            image: ubuntu:20.04
           - distro: rolling
             branch: main
             image: ubuntu:jammy
-          - distro: galactic
-            branch: galactic
-            image: ubuntu:20.04
+          - distro: iron
+            branch: release/iron
```

```suggestion
            branch: iron
```

Modify this before merge